people.json

```diff
@@ -0,0 +1 @@
+[{"json_class":"Student","id":null,"name":null,"age":{"name":"Rahyan","age":"12","parent_permission":false},"parent_permission":null}]
```

Comment:
- [ ] This is an example of `people.json`:
`[{"json_class":"Student","name":"Sam","age":"14","parent_permission":true},{"json_class":"Student","name":"Pom","age":"16","parent_permission":false},{"json_class":"Teacher","name":"Fox","age":"56","parent_permission":true,"specialization":"Starships"},{"json_class":"Student","name":"Poppy","age":"20","parent_permission":false}]`